index.html

```diff
@@ -671,16 +671,36 @@ <h4>OAuth 2.0</h4>
     <section>
       <h3>Service Instances</h3>
       <p>
-Service configuration is determined by administrators on per instance basis.
-Instance configuration can include, but is not limited to, credential format, key options, credential status mechanisms, and/or credential templates.
-Administrators are expected to configure service instances such that `options` included in client requests cannot result in incorrect action or problematic responses by the service.
+The APIs defined in this specification presume that they are attached to a
+specific instance of some kind with an associated configuration that has been
+put in place by a systems administrator. When a client calls an endpoint on a
+particular instance, the instance uses the configuration and options provided by
+the client to execute the action.
       </p>
       <p>
-A coordinator instance can have access to multiple service instances in order to support different use cases.
-Runtime discovery of service instance configuration is not supported by the VC API as services are expected to be known by the coordinator at the time of coordinator deployment.
+For example, the `/credentials/issue` endpoint is expected to be at the end of a
+longer URL such as `/instances/12345/credentials/issue`. In this case, it is the
```

Comment:
Does it need to be at the end of longer URL? It seems reasonable to me that one could direct to specific instances in other ways. For example, one could use subdomains to address an instance or an internal service discovery mechanism.

Reply:
+1, I made a suggestion in my review.